core/src/banking_stage.rs

```diff
@@ -1020,8 +1021,10 @@ mod tests {
             let blocktree = Arc::new(
                 Blocktree::open(&ledger_path).expect("Expected to be able to open database ledger"),
             );
+            let mut poh_config = PohConfig::default();
+            poh_config.target_tick_count = Some(6);
```

Comment:
For clarity, instead of the magic number 6, could you write this as `genesis_block.ticks_per_slot + num_extra_ticks` for some number `num_extra_ticks`.

Reply:
sure thing! done at https://github.com/solana-labs/solana/pull/6251/commits/a9138d1efdcaff8d1b8979c638c13aefab0ed444

Reply:
You were right 😄 for the same reasons as the other cases listed below, this should also use `bank.max_tick_height()` instead of `genesis_block.ticks_per_slot` for clarity and consistency

Reply:
Nice catch! done cef4770bee4c27a70fe4001ae35dae29f148e984